svd.py

```diff
@@ -1,226 +1,287 @@
+'''Golub-Reinsch SVD algorithm
+     This implementation is based on:
+     "Numerical Recipes in C" section 2.6, 
+     see http://www.grad.hr/nastava/gs/prg/NumericalRecipesinC.pdf
+'''
```

Comment:
Please add an author attribution here, you can link to your GitHub or Upwork profile if you want

Reply:
Sure! I am going to do it!